### ISSUE TITLE
Site: broken link

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [vimalloc/flask-jwt-extended#267]
1. here on site:
https://flask-jwt-extended.readthedocs.io/en/latest/installation.html

2. click on `Stable`:
```
The latest stable version is Stable.
```

3. comes back with error:
https://flask-jwt-extended.readthedocs.io/en/latest/stable/installation.html
```
Page Not Found
The page you requested does not exist. You may have followed a bad link, or the page may have been moved or removed.
```

**Comment by vimalloc** [comment]
Thanks for the report. I’m not sure if that is a result of a read the docs bug or something to do with migrating to the new Sphinx theme in this commit https://github.com/vimalloc/flask-jwt-extended/commit/f39a679b57cae9198dafab174a028febef97802a. I’ll try to dig into this some more when I have some time.

**Comment by vimalloc** [comment]
The broken link seems to be something to do with pallets-sphinx-theme. I still need to dig into that more, but in the mean time I changed the default page to be the `/stable` version, which should hopefully alleviate the issue a bit.

**Comment by DarkSuniuM** [comment]
> The broken link seems to be something to do with pallets-sphinx-theme. I still need to dig into that more, but in the mean time I changed the default page to be the `/stable` version, which should hopefully alleviate the issue a bit.

@vimalloc 
I tried it with other themes, the problem remains,
Here is a work around

0. Go to your project's dashboard on readthedocs.io
0. Go to admin
0. Go to advanced settings
0. Change  Documentation type to Shpinx HtmlDir
0. Go to builds
0. Build for both `stable` and `latest` manually

This will fix the problem by changing the URLs
I tried it with this documentations on this URL: https://darksun-flask-jwt-extended.readthedocs.io/en/latest/installation/
